README.md

```diff
@@ -81,7 +81,7 @@ Finally, existing bundlers are built around string loaders/transforms, where the
 
 `parcel` transforms a tree of assets to a tree of bundles. Many other bundlers are fundamentally based around JavaScript assets, with other formats tacked on - for example, by default inlined as strings into JS files. `parcel` is file-type agnostic - it will work with any type of assets the way you'd expect, with no configuration.
 
-`parcel` takes as input a single entry asset, which could be any type: a JS file, HTML, CSS, image, etc. There are various asset types defined in `parcel` which know how to handle specific file types. The assets are parsed, their dependencies are extracted, and they are transformed to their final compiled form. This creates a tree of assets.
+`parcel` takes as input a single entry asset, which could be any file type: JS, HTML, CSS, image(PNG/JPEG), etc. There are various asset types defined in `parcel` which know how to handle specific file types. The assets are parsed, their dependencies are extracted, and they are transformed to their final compiled form. This creates a tree of assets.
```

Comment:
This makes it seem like only JPEG / PNG images are supported.

Reply:
Great point, I've removed `PNG/JPEG`,